### ISSUE TITLE
example banned.txt

### ISSUE THREAD

**Issue by jianlins** [rtyley/bfg-repo-cleaner#514]
Can you provide an example of banned.txt. Despite the brief introduction in the command line hit, there is still some ambiguous part, e.g. is a whitespace needed or ignorable before and after "==>", how to place prefixes, etc.

**Comment by SoundsSerious** [comment]
Yes this would be great! I also found the installation a bit confusing, it would be great to show how to use an alias to use bfg as in the examples as well

**Comment by arpan3t** [comment]
#154 [Comment](https://github.com/rtyley/bfg-repo-cleaner/issues/154#issuecomment-221590263)
